### PR TITLE
Ax 400 athlete graph

### PR DESCRIPTION
**Pull request by pearlson16**
# Description
Query API for Book Value over a time range. Plot the values on the graph.

Fixes Jira Ticket # https://athletex.atlassian.net/browse/AX-400

## Type of change
Please delete options that are not relevant.
- [x] Bug fix (non-breaking change which fixes an issue)

# Is this a UI Change? If so please include screenshot of before and after states below

image
image

# Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have assigned 2 reviewers to check my work
- [x] My changes generate no new warnings

### CODE REVIEW

File: lib/pages/athlete/models/AthletePageState.dart [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  AthletePageState({
+    required this.stats,
+    required this.status,
+  });
+
+  // const AthletePageState(
+  //     {this.status = BlocStatus.initial, MLBAthleteStats? stats})
+  //     : this.stats = stats ??
+  //           const MLBAthleteStats(
+  //               id: 0, name: "", team: "", position: "", statHistory: []);
```
Comment (Friendlygeckos) [inline lib/pages/athlete/models/AthletePageState.dart:17]: Remove this if we do not need it

File: lib/pages/athlete/bloc/AthletePageBloc.dart [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  }
+
+  void _mapPageRefreshEventToState(
+      OnPageRefresh event, Emitter<AthletePageState> emit) async {
+    try {
+      print("load page stats");
+      final int playerId = event.playerId;
+      emit(state.copyWith(status: BlocStatus.loading));
+      final MLBAthleteStats stats = await repo.getPlayerStatsHistory(
+          playerId, "2022-06-01", "2022-06-30");
```
Comment (Friendlygeckos) [inline lib/pages/athlete/bloc/AthletePageBloc.dart:25]: Lets not hardcode the dates in. We can try to get the current date and the older date and then pass them in here.

**Review by Friendlygeckos — CHANGES_REQUESTED** [review]

File: lib/pages/athlete/bloc/AthletePageBloc.dart (hunk 2 above)
Comment (pearlson16) [reply lib/pages/athlete/bloc/AthletePageBloc.dart:25]: done, as for the older date there is no way to get that

File: lib/pages/athlete/models/AthletePageState.dart (hunk 1 above)
Comment (pearlson16) [reply lib/pages/athlete/models/AthletePageState.dart:17]: done

**Review by MichaelFrank0317 — APPROVED** [review]

**Pull request merged**